### ISSUE TITLE
Feature: clear_notification to remove notification

### ISSUE THREAD

**Issue by disrupted** [home-assistant/iOS#579]
**Is your feature request related to a problem? Please describe.**
the Android companion already has this feature ([`clear_notification`](https://companion.home-assistant.io/docs/notifications/notifications-basic/#replacing-notifications)). When sending push notifications to one or multiple devices and circumstances change you might want to remove a previously sent notification. In my case I am sending an actionable notification to all my devices. After responding on one of my devices (or in case the circumstances change) I'd like to clear the notification on all my other devices.

**Describe the solution you'd like**
expose a service to silently dismiss/remove notifications

**Describe alternatives you've considered**
none

**Additional context**
This is something we're all familiar with from messaging apps. Read a message on one device, the notifications get cleared on all other devices.

**Comment by deluxestyle** [comment]
you can already do this. 
```
data:
      message: clear_badge
service: notify.mobile_app_iphone
```

**Comment by airdrummingfool** [comment]
@deluxestyle is there a way to provide the `thread-id` of the messages, in order to remove only certain notifications?

**Comment by deluxestyle** [comment]
Sorry. I just use the clear badge. I even cannot remember where I found that

**Comment by boesing** [comment]
@airdrummingfool You probably have to use a `apns-collapse-id` to do this. Its documented in the documentation: https://companion.home-assistant.io/docs/notifications/notifications-basic/#thread-id-grouping-notifications

**Comment by niccolobusato** [comment]
+ 1 for that

**Comment by fmonday** [comment]
Thanks for that @deluxestyle... the documentation says to use `delete_alert` as the message; that didn't work for me, but `clear_badge` did.

**Comment by skrenes** [comment]
Hmm. I tried both `delete_alert` and `clear_badge` and neither worked to remove the notification. Incidentally, unlike any other other message, including _clear_notification_, those two keywords don't generate a notification or change the message in the existing notification. I do hope we can get this feature. [I suggested a way in the aforementioned PR](https://github.com/home-assistant/iOS/pull/587#issuecomment-641343474).
